lib/classes/CLI.test.js

```diff
@@ -302,6 +305,72 @@ describe('CLI', () => {
     });
   });
 
+  describe('#logBreakingChanges()', () => {
+    let consoleLogStub;
+
+    beforeEach(() => {
+      cli = new CLI(serverless);
+      consoleLogStub = sinon.stub(cli, 'consoleLog').returns();
+    });
+
+    afterEach(() => {
+      cli.consoleLog.restore();
+      delete process.env.SLS_IGNORE_WARNING;
+    });
+
+    it('should log breaking changes when they are provided', () => {
+      const nextVersion = 'Next';
+
+      cli.breakingChanges = [
+        'x is broken',
+        'y will be updated',
+      ];
+
+      let expectedMessage = '\n';
+      expectedMessage += chalk.yellow(`  WARNING: You are running v${serverlessVersion}. v${nextVersion} will include the following breaking changes:\n`);; //eslint-disable-line
+      expectedMessage += chalk.yellow('  - x is broken\n');
+      expectedMessage += chalk.yellow('  - y will be updated\n');
+      expectedMessage += '\n';
+      expectedMessage += chalk.yellow('  You can opt-out from these warnings by setting the "SLS_IGNORE_WARNING=*" environment variable.\n'); //eslint-disable-line
```

Comment:
Yes, I agree. checking everything in detail might be too detailed 👍 

Maybe for future situations we can check e.g. that the error / warning message contains the Breaking change items etc. so that we don't have to go through hell when we change the message itself 👹